Missing WebCrypto APIs from webworker.d.ts

> tsc --lib webworker,es2015 a.ts

```txt
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(964,35): error TS2304: Cannot find name 'Algorithm'.
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(970,11): error TS2304: Cannot find name 'AlgorithmIdentifier'.
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(973,35): error TS2304: Cannot find name 'KeyAlgorithm'.
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(979,11): error TS2304: Cannot find name 'AlgorithmIdentifier'.
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(983,11): error TS2304: Cannot find name 'AlgorithmIdentifier'.
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(990,33): error TS2304: Cannot find name 'Algorithm'.
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(994,31): error TS2304: Cannot find name 'Algorithm'.
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(995,11): error TS2304: Cannot find name 'AlgorithmIdentifier'.
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(998,34): error TS2304: Cannot find name 'Algorithm'.
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(1002,34): error TS2304: Cannot find name 'KeyAlgorithm'.
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(1010,39): error TS2304: Cannot find name 'Algorithm'.
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(1011,13): error TS2304: Cannot find name 'CryptoKey'.
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(1014,32): error TS2304: Cannot find name 'Algorithm'.
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(1019,35): error TS2304: Cannot find name 'KeyAlgorithm'.
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(1023,35): error TS2304: Cannot find name 'Algorithm'.
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(1027,39): error TS2304: Cannot find name 'Algorithm'.
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(1031,32): error TS2304: Cannot find name 'Algorithm'.
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(1035,33): error TS2304: Cannot find name 'Algorithm'.
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(1039,32): error TS2304: Cannot find name 'Algorithm'.
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(1045,32): error TS2304: Cannot find name 'Algorithm'.
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(1049,36): error TS2304: Cannot find name 'Algorithm'.
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(1050,12): error TS2304: Cannot find name 'AlgorithmIdentifier'.
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(1054,36): error TS2304: Cannot find name 'KeyAlgorithm'.
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(1055,11): error TS2304: Cannot find name 'AlgorithmIdentifier'.
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(1059,36): error TS2304: Cannot find name 'Algorithm'.
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(1060,11): error TS2304: Cannot find name 'AlgorithmIdentifier'.
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(1064,34): error TS2304: Cannot find name 'Algorithm'.
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(1069,34): error TS2304: Cannot find name 'KeyAlgorithm'.
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(1074,37): error TS2304: Cannot find name 'Algorithm'.
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(1075,13): error TS2304: Cannot find name 'CryptoKey'.
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(1078,37): error TS2304: Cannot find name 'Algorithm'.
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(1083,32): error TS2304: Cannot find name 'Algorithm'.
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(1084,12): error TS2304: Cannot find name 'AlgorithmIdentifier'.
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(1092,33): error TS2304: Cannot find name 'Algorithm'.
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(1093,11): error TS2304: Cannot find name 'AlgorithmIdentifier'.
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(1098,32): error TS2304: Cannot find name 'Algorithm'.
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(1101,11): error TS2304: Cannot find name 'AlgorithmIdentifier'.
../Users/mhegazy/AppData/Roaming/npm/node_modules/typescript/lib/lib.webworker.d.ts(1200,24): error TS2304: Cannot find name 'WheelEvent'.
```

ran into this during investigating https://github.com/Microsoft/TypeScript/issues/9244

--- a/src/lib/webworker.generated.d.ts
+++ b/src/lib/webworker.generated.d.ts
@@ -3,6 +3,10 @@
 /// IE Worker APIs
 /////////////////////////////
 
+interface Algorithm {
+    name: string;
+}
+
 interface EventInit {
     bubbles?: boolean;
     cancelable?: boolean;
@@ -16,6 +20,10 @@ interface IDBIndexParameters {
 interface IDBObjectStoreParameters {
     autoIncrement?: boolean;
     keyPath?: IDBKeyPath;
+}
+
+interface KeyAlgorithm {
+    name?: string;
 }
 
 interface EventListener {
@@ -105,6 +113,18 @@ interface Coordinates {
 declare var Coordinates: {
     prototype: Coordinates;
     new(): Coordinates;
+}
+
+interface CryptoKey {
+    readonly algorithm: KeyAlgorithm;
+    readonly extractable: boolean;
+    readonly type: string;
+    readonly usages: string[];
+}
+
+declare var CryptoKey: {
+    prototype: CryptoKey;
+    new(): CryptoKey;
 }
 
 interface DOMError {
@@ -1178,7 +1198,7 @@ declare var console: Console;
 declare function addEventListener(type: "error", listener: (ev: ErrorEvent) => any, useCapture?: boolean): void;
 declare function addEventListener(type: "message", listener: (ev: MessageEvent) => any, useCapture?: boolean): void;
 declare function addEventListener(type: string, listener: EventListenerOrEventListenerObject, useCapture?: boolean): void;
+type AlgorithmIdentifier = string | Algorithm;
 type IDBKeyPath = string;
 type IDBValidKey = number | string | Date | IDBArrayKey;
 type BufferSource = ArrayBuffer | ArrayBufferView;
-type MouseWheelEvent = WheelEvent;

